### PR TITLE
added remove user logic

### PR DESCRIPTION
**Pull request by fryette**

### CODE REVIEW

File: Softeq.XToolkit.Chat.iOS/ViewControllers/ChatDetailsViewController.cs [hunk 1]
```diff
@@ -57,6 +60,30 @@ protected override void DoAttachBindings()
...
+            private readonly ChatDetailsViewModel _viewModel;
+
+            public ParticipantsTableViewDelegate(ChatDetailsViewModel viewModel)
+            {
+                _viewModel = viewModel;
+            }
+
+            public override UITableViewRowAction[] EditActionsForRow(UITableView tableView, NSIndexPath indexPath)
+            {
+                if (_viewModel.IsEditAvailable && !_viewModel.IsMemberRemovable((int) indexPath.Item))
```
Comment (wcoder) [inline Softeq.XToolkit.Chat.iOS/ViewControllers/ChatDetailsViewController.cs]: you are right, move `IsEditAvailable` to VM

File: Softeq.XToolkit.Chat.iOS/ViewControllers/ChatDetailsViewController.cs [hunk 2]
```diff
@@ -57,6 +60,30 @@ protected override void DoAttachBindings()
...
         }
-    }
-}
 
+        private class ParticipantsTableViewDelegate : UITableViewDelegate
+        {
+            private readonly ChatDetailsViewModel _viewModel;
+
+            public ParticipantsTableViewDelegate(ChatDetailsViewModel viewModel)
+            {
```
Comment (wcoder) [inline Softeq.XToolkit.Chat.iOS/ViewControllers/ChatDetailsViewController.cs]: depends on VM lifetime (register in IoC), use weakRef of VM

Comment (fryette) [reply Softeq.XToolkit.Chat.iOS/ViewControllers/ChatDetailsViewController.cs]: discussed

File: Softeq.XToolkit.Chat.iOS/ViewControllers/ChatDetailsViewController.cs (hunk 1 above)
Comment (fryette) [reply Softeq.XToolkit.Chat.iOS/ViewControllers/ChatDetailsViewController.cs]: fixed

**Pull request merged**